### DISCUSSION TITLE
Time-consuming

### DISCUSSION THREAD

**Discussion by zhhl9101** [openai/whisper#837] [Q&A] [answered]
I put 28 audios to service, total 340s, but it took 1945s to get the result, is this normal? 
Are there any settings or parameters that can improve the recognition speed?
Used "large" model version for Chinese.

**Comment by glangford** [comment, marked as answer]
It sounds like you are running on CPU 
* if you have a supported GPU, make sure `whisper` is using it (see for example https://github.com/openai/whisper/discussions/603)
* to transcribe multiple videos at once, see https://github.com/openai/whisper/discussions/153 and for 28 files, you can just put all of them on the command line

See the following past discussions
* https://github.com/openai/whisper/discussions/212
* https://github.com/openai/whisper/discussions/716
* https://github.com/openai/whisper/discussions/208
* https://github.com/openai/whisper/discussions/530
* https://github.com/openai/whisper/discussions/662

**Reply by glangford** [reply]
Also
* https://github.com/openai/whisper/discussions/391
